src/pytorch_lightning/strategies/colossalai.py

```diff
@@ -1,17 +1,19 @@
-from typing import Any, Generator, Optional
+from typing import Any, Dict, Generator, List, Optional
 
 import torch
-from lightning_utilities.core.imports import RequirementCache
 
 import pytorch_lightning as pl
 from pytorch_lightning.accelerators.cuda import CUDAAccelerator
 from pytorch_lightning.overrides.base import _LightningModuleWrapperBase, _LightningPrecisionModuleWrapperBase
-from pytorch_lightning.plugins.precision import ColossalAIPrecisionPlugin
+from pytorch_lightning.plugins.environments.cluster_environment import ClusterEnvironment
+from pytorch_lightning.plugins.io.checkpoint_plugin import CheckpointIO
+from pytorch_lightning.plugins.precision import ColossalAIPrecisionPlugin, PrecisionPlugin
 from pytorch_lightning.strategies.ddp import DDPStrategy
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.imports import _RequirementAvailable
 from pytorch_lightning.utilities.types import STEP_OUTPUT
 
-_COLOSSALAI_AVAILABLE = RequirementCache("colossalai")
```

Comment:
why not using `RequirementCache`?

Reply:
Since the previous version didn't merge with the master branch, I have to use the old version of RequirementAvail to run the test. I have changed it to RequirementCache now.